charts/lenses/templates/ingress.yaml

```diff
@@ -17,21 +17,25 @@ metadata:
   {{- end }}
 spec:
   rules:
-  - host: {{ .Values.ingress.host }}
+  {{- range .Values.ingress.hosts }}
```

Comment:
what's the goal you are trying to achieve with multiple hosts? Because it's a breaking change and I think we can treat this only with a minor bump of the version and have backward compatible helm charts.

Reply:
My need was to basically add CNAMEs. I wanted to deploy an API into production and make sure everything was working before replacing the current API. For this I would need two hosts as follows: `api.prod.example.com ` and `api.example.com`

Reply:
ok got it. let's have this with a backward compatible way then.

Reply:
sounds good, should be fine now